Library/.rubocop.yml

```diff
@@ -28,7 +28,7 @@ FormulaAuditStrict/Options:
   Enabled: true
 
 NewFormulaAudit/Options:
-  Enabled: false
+  Enabled: true 
```

Comment:
This still won't quite work because it'll show up in files when using e.g. `rubocop` in my editor. Need some way to have it be disabled by default and enabled only when requested.